package.json

```diff
@@ -40,12 +40,12 @@
   "engineStrict": true,
   "dependencies": {
     "@interlay/esplora-btc-api": "0.4.0",
-    "@interlay/interbtc-types": "0.10.2",
-    "@interlay/monetary-js": "0.2.2",
+    "@interlay/interbtc-types": "0.12.3",
+    "@interlay/monetary-js": "0.2.4",
```

Comment:
Could you upgrade to the monetary lib version that already has the fix for the `DOTAmount` in it to make the naming across currencies consistent please?

Reply:
On naming, would it be nicer to properly camelCase some of these types? i.e. `DotAmount`

Reply:
I prefer the ticker-style naming. But that's personal preference